.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "contracts/lib/forge-std"]
 	path = contracts/lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
+	url = https://github.com/foundry-rs/forge-std@v1.11.0
 [submodule "contracts/lib/openzeppelin-contracts"]
 	path = contracts/lib/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts@v5.5.0
```

Comment:
I am against this change, as it makes submodule management a hassle and easy to break out of sync.

The version of the submodule is already tracked by git - there is no need to include versions here in the `.gitmodules` file. I am not sure what problems you were running into with Foundry, but just doing a `git submodule update --recursive` should make everything work.

I actually don't like the whole `forgelock` file that they added precisely because there is now competing dependency management: git submodules and forge dependencies. I would prefer we just stick to Git (unless it starts breaking Foundry tooling).

Reply:
This issue was resolved based on the above comment and an internal discussion.